transactor/src/test/scala/io/mediachain/copycat/DummyContext.scala

```diff
@@ -16,16 +16,16 @@ object DummyContext {
     println("*** SETUP DUMMY COPYCAT CONTEXT")
     val logdir = setupLogdir()
     val store = new Dummies.DummyStore
-    val server = Copycat.Server.build(address, logdir, store, blocksize)
+    val server = Server.build(address, logdir, store, None, blocksize)
     server.bootstrap().join()
-    val client = Copycat.Client.build()
+    val client = Client.build()
     client.connect(address)
     DummyContext(server, client, store, logdir)
   }
   
   def setupLogdir() = {
     import sys.process._
-    val logdir = "mktemp -d".!!
+    val logdir = "mktemp -d".!!.trim
```

Comment:
not sure what tg was seeing, but this was a facepalm: the logdir had an extra newline in the end and the rm -rf failed quietly because of this. The result: /tmp/tmp.xxx garbage left behind.